inspirehep/utils/citations.py

```diff
@@ -61,3 +61,23 @@ def get_and_format_citations(record):
         result.append(row)
 
     return result
+
+
+def get_citations_from_es(record, page=1, size=10):
+    if 'control_number' not in record:
+        return []
```

Comment:
my bad, it's better to return ``None``. It doesn't affect the api because we handle the NotExistingRecord cases.